internal/cfgplugins/policyforwarding.go

```diff
@@ -570,6 +571,56 @@ func aristaGreDecapCLIConfig(t *testing.T, dut *ondatra.DUTDevice, params OcPoli
 
 }
 
+func InterfacePolicyForwardingApply(t *testing.T, dut *ondatra.DUTDevice, interfaceName string, policyName string, ni *oc.NetworkInstance, params OcPolicyForwardingParams) {
```

Comment:
This plugin function does not follow the proper style, see https://github.com/openconfig/featureprofiles/blob/main/internal/cfgplugins/README.md.   All config parameters must be in a struct